### DISCUSSION TITLE
為何 ChatGPT 給出的答案會有錯誤？

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#657] [General]
因為上一篇提到Stack Overflow 禁止分享 ChatGPT 產生的答案因為錯誤率太高
所以我上網找了一下，找到Inside訪問專攻自然語言處理和對話系統的台灣大學資訊工程學系陳縕儂副教授的專訪：
其中陳縕儂副教授提到ChatGPT 給出的答案之所以會有錯誤 「是因為雖然 ChatGPT 在第一步驟會盡可能收集大量的人類回答，但由於問題和答案種類有無限多，ChatGPT 模型無法涵蓋所有對話資料，且其抓取資料的時間範圍為今年前，因此模型無法正確答出較新的資訊。另外，AI 自動生成模型普遍會產生幻覺（hallucination），當給出兩筆不具細節且彼此關連性較弱的資訊時，ChatGPT 就容易自行補上未必正確的細節。」

這邊專訪中還有許多關於ChatGPT的解答 歡迎大家點進去看喔！
網址：https://www.inside.com.tw/article/30190-chatgpt-professor-Vivian-Chen-interview
另外 我高中時有參加台灣傑出女科學家獎的頒獎典禮 我去的那一年剛好陳縕儂副教授有上台受獎喔 那時候覺得她又聰明又漂亮

**Comment by Ranger22343** [comment]
我周遭的人給出的規律是，定性描述(意思是針對性質的描述)chatGPT可以做得不錯，但是定量描述(意思是針對事實統計與計算的描述)就很容易出問題，就算是很簡單的數學題目也是。

**Reply by liangyinglly** [reply]
謝謝你精闢的解說
